app/src/test/java/com/main/mealplanner/RecipeUnitTests.kt

```diff
@@ -1,26 +1,22 @@
 package com.main.mealplanner
 
-import androidx.lifecycle.MutableLiveData
-import org.junit.Test
 import com.main.mealplanner.dto.Recipe
-import com.main.mealplanner.service.RecipeService
-import org.junit.Assert.*
 import io.mockk.confirmVerified
-import io.mockk.every
-import io.mockk.mockk
 import io.mockk.verify
+import org.junit.Assert.*
 import org.junit.Rule
+import org.junit.Test
 import org.junit.rules.TestRule
 
 class RecipeUnitTests {
-
+    //TODO() Insure that unit tests will work for the future
```

Comment:
Putting conversation/discussion  in code comments is bad practice, as that clutters up the source code and does not belong in source code files.  I mentioned that in the code review instructions.    Discussion and comments are very good, but have that in the appropriate place, which is in the "Files Changed" tab in the pull request in GitHub (which is where I am writing this and you are reading this).  That way, you can have a conversation with replies, right where the code is happening, and without cluttering up the source code itself.